src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,42 @@
+.calendar {
+  width: $day-size * 7 + $day-gap * 6;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+
+  &__day {
+    width: $day-size;
+    height: $day-size;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 1px solid #000;
+    background-color: #eee;
+    transition-duration: 500ms;
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: pink;
+      cursor: pointer;
+    }
```

Comment:
`cursor: pointer;` should be moved to `&__day`

Reply:
Ok, moved  &__day

Reply:
Is it ok how I write this selector?
"&--start-day_#{$day} :first-child {...}"

and how about this ?
"&--month-length_#{$date} :nth-child(n + #{$date + 1}) {...}"

I mean space in the middle for nesting.

Reply:
please, feel free to ask it in the slack chat